lnbits/core/models/__init__.py

```diff
@@ -73,6 +74,7 @@
     "Account",
     "AccountFilters",
     "AccountOverview",
+    "UserACLs",
```

Comment:
maybe, `UserAcls` for keeping consistent with upper camelcase?

Reply:
done